test/Resources/config/services.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Metadata\BlackHole"
+                 class="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Metadata\BlackHole"/>
+        <service id="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Plugin\BlackHole"
+                 class="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Plugin\BlackHole"/>
+        <service id="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Adapter\BlackHole"
+                 class="ProophTest\Bundle\EventStore\DependencyInjection\Fixture\Adapter\BlackHole"/>
+    </services>
```

Comment:
I need this service configuration for testing only. Any ideas how to inject them, so they are available in the bundle?